config/locales/hyrax.en.yml

```diff
@@ -727,6 +730,7 @@ en:
         keyword:                   "Keyword"
         lease_expiration_date:     'until'
         related_url:               "Related URL"
+        title:                     "Title"
```

Comment:
Let's say admin set stops using the title field altogether.  At this point I know that I can remove the translation for `admin_set.title`, however, if it's in `default.title`, I don't know that it can be safely removed.  So putting it somewhere generic, like default, prevents us from doing clean up down the road.